src/lib/Scenes/Artwork/Components/CommercialInformation.tsx

```diff
@@ -14,19 +15,26 @@ interface CommercialInformationProps {
   artwork: CommercialInformation_artwork
 }
 
+export type AuctionState = "hasStarted" | "isLive" | "hasEnded" | "isPreview" | null
```

Comment:
We'd talked before about not allowing this to be null and instead making sure we're correctly setting the initial state. I _think_ that would be preferable since `null` doesn't have any meaning in this case.